tests/modules/programs/tmux/emacs-with-plugins.nix

```diff
@@ -0,0 +1,35 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  substituteExpected = path: pkgs.substituteAll {
+    src = path;
+
+    sensible_rtp = pkgs.tmuxPlugins.sensible.rtp;
```

Comment:
@rycee on Travis, this is returning a different path than the one used by the module, do you know why?

Reply:
Hmm, from what I can tell in the [Travis run](https://travis-ci.org/rycee/home-manager/jobs/491299359#L1694-L1719) they are the same. Perhaps I'm missing something? The assertion failure I see is
```
Expected home-files/.tmux.conf to be same as /nix/store/kn7lw7l4gi0x5dchplzrlf9yr4p5xvl6-emacs-with-plugins.conf but were different:
--- actual
+++ expected
@@ -35,18 +35,18 @@
 # tmuxplugin-logging
 # ---------------------
 
-run-shell /nix/store/w8zrkxr29371f1pwj060n310mf21shby-tmuxplugin-logging/share/tmux-plugins/logging/logging.tmux
+run-shell /nix/store/ppyx7z9m8n9dkq3p977r8f0sv20rynaf-tmuxplugin-logging/share/tmux-plugins/logging/logging.tmux
 
 
 # tmuxplugin-prefix-highlight
 # ---------------------
 
-run-shell /nix/store/agzl590w3aw985lijvqyv5xjdw7c5zlf-tmuxplugin-prefix-highlight/share/tmux-plugins/prefix-highlight/prefix_highlight.tmux
+run-shell /nix/store/3i3hb0sxn11cf47rbnq4vcil2bpsjiiy-tmuxplugin-prefix-highlight/share/tmux-plugins/prefix-highlight/prefix_highlight.tmux
 
 
 # tmuxplugin-fzf-tmux-url
 # ---------------------
 
-run-shell /nix/store/hjcbcnah3j8a4ikd6dfr80gcdmm0qq5m-tmuxplugin-fzf-tmux-url/share/tmux-plugins/fzf-tmux-url/fzf-url.tmux
+run-shell /nix/store/hhkdfsqdbdd2kagzp1yfbjna8qh77gws-tmuxplugin-fzf-tmux-url/share/tmux-plugins/fzf-tmux-url/fzf-url.tmux
 
 # ============================================= #
```
which doesn't include the sensible plugin.